fr-index.html

```diff
@@ -0,0 +1,64 @@
+---
+title: "Jamulus ‒ Jouer gratuitment et librement de la musique en ligne avec des ami·e·s."
+lang: "fr"
+permalink: /
+layout: mainhomepage
+ldjAppCategory: "Musique"
+ldjSlogan: "Jouer gratuitment et librement de la musique en ligne avec des ami·e·s."
+ldjCreator: "Volker Fischer et contributeurs"
+metadescription: "Jamulus est un logiciel libre qui permet aux musicien·ne·s de jouer de la musique, de répéter ou d'improviser avec d'autres musicien·ne·s, le tout en temps réel sur Internet."
+mAltProgIcon: "Icône Jamulus"
+mTSlogan: "Jouer gratuitment et librement de la musique en ligne avec des ami·e·s."
+mTGetStartedNow: "Démarrer"
+mHTMLDownloadNow: '<a href="https://sourceforge.net/projects/llcon/files/latest/download" target="_blank" id="dld_frm_sf" rel="noreferrer">Télécharger</a>'
+---
+<div class="fx-row fx-row-center-xs" id="firstrow">
+	<div class="fx-col-100-xs">
+		<div itemprop="abstract">
+			<h2>Qu'est-ce que Jamulus&nbsp;?</h2>
+			<p>
+			Jamulus est fait pour jouer, répéter, ou simplement improviser avec vos ami·e·s, votre groupe ou n'importe qui que vous trouverez en ligne.
+			Jamulus est multi plateforme, vous pouvez l'utilisez avec votre machine sous Windows, macOS ou Linux pour vous connecter aux serveurs Jamulus publics et privés du monde entier, ce gratuitement et librement, via une connexion Internet haut-débit.<br/>
+			Vous pouvez également administrer et héberger <a href="wiki/Running-a-Server" target="_blank" rel="noreferrer">votre propre serveur</a> public ou privé.<br/>
+			Jamulus est conçu pour offrir un son de haute qualité et à faible latence, ce qui permet de jouer ensemble à distance et en temps réel.<br/>
+			Regardez <a href="wiki/Demos" target="_blank" rel="noreferrer">comment les gens s'y prennent</a>&nbsp;!
+			</p>
+		</div>
+	</div>
+</div>
+<div class="fx-row fx-row-center-xs" id="bannercontainer">
+	<div class="fx-col-100-xs">
+		<a href="wiki/Getting-Started" target="_blank" rel="noreferrer">
+			<img alt="Jamulus Banner. Links to getting started page" src="{{ '/assets/img/jamulusbannersmall.png' | relative_url }}" id="jamulusbanner" loading="lazy" />
```

Comment:
Thanks!

Could you please also translate alt texts?

Reply:
Done! I haven't see this.